statsd/statsd.go

```diff
@@ -285,8 +285,8 @@ func (c *Client) Set(name string, value string, tags []string, rate float64) err
 }
 
 // Timing sends timing information, it is an alias for TimeInMilliseconds
-func (c *Client) Timing(name string, value float64, tags []string, rate float64) error {
-	return c.TimeInMilliseconds(name, value, tags, rate)
+func (c *Client) Timing(name string, value time.Duration, tags []string, rate float64) error {
+	return c.TimeInMilliseconds(name, float64(value.Nanoseconds())/float64(time.Millisecond), tags, rate)
```

Comment:
I think @dsansotscea is probably right it does read better, and is probably a little faster. That said this has been waiting for a long time, so if @gphat can't address this in a day or so, I'll merge and fix that up myself in a separate PR.

Thank you both!